js/index.js

```diff
@@ -141,29 +141,27 @@ Raven.context(async () => {
     document.getElementById("app").style.visibility = "hidden";
     return;
   }
-  const __extraWindows = [];
-  let __initialWindowLayout = null;
 
+  let __butterchurnOptions = null;
+  let __initialWindowLayout = null;
   if (isButterchurnSupported()) {
     const startWithMilkdropHidden =
+      library ||
       document.body.clientWidth < MIN_MILKDROP_WIDTH ||
       (!library && skinUrl != null) ||
       screenshot;
 
-    __extraWindows.push({
-      id: "milkdrop",
-      title: "Milkdrop",
-      isVisualizer: true,
-      Component: MilkdropWindow,
-      open: !library && !startWithMilkdropHidden
-    });
+    __butterchurnOptions = Object.assign(
```

Comment:
Nit: Everywhere else in the code base we use the object spread operator. I should add a lint rule to enforce that or something.

Reply:
sounds good